docs/config-ha.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Configuring high-availability components"
+linkTitle: "Configuring high-availability components"
+weight: 20
+type: "docs"
+---
+
+Active/passive high availability (HA) is a standard feature of Kubernetes APIs that helps to ensure that APIs stay operational if a disruption occurs. In an HA deployment, if an active controller crashes or is deleted, another controller is available to take over processing of the APIs that were being serviced by the controller that is now unavailable.
+
+Active/passive HA in Knative is available through leader election, which can be enabled after Knative Serving control plane is installed.
+
+When using a leader election HA pattern, instances of controllers are already scheduled and running inside the cluster before they are required. These controller instances compete to use a shared resource, known as the leader election lock. The instance of the controller that has access to the leader election lock resource at any given time is referred to as the leader.
+
+HA functionality is available on Knative for the `autoscaler-hpa`, `controller` and `activator` components.
```

Comment:
This seems inconsistent with the list below?

Reply:
Perhaps we should include a list of what's supported *and not supported* broken down by namespace?

As a user, I'd probably want to know what isn't HA (yet).

Reply:
@pmorie can you provide details for this please?

Reply:
I'm honestly not sure if including a list of components that are or aren't supported here makes sense if that list is likely to be in flux for now, docs maintenance wise. I'll add it, but let's revisit and discuss the future plans for HA docs soon.